inspirehep/modules/arxiv/__init__.py

```diff
@@ -26,4 +26,6 @@
 
 from .ext import Arxiv
 
-__all__ = ('Arxiv')
+__all__ = [
```

Comment:
It seems like a mistake and the correct syntax to have a tuple should be `('Arxiv',)`